etl-pipeline/processes/ingest/gutenberg.py

```diff
@@ -117,16 +107,16 @@ def add_cover(self, gutenberg_record: GutenbergMapping):
             file_type = re.search(r'(\.[a-zA-Z0-9]+)$', cover_data.get('image_path')).group(1)
             cover_path = 'covers/gutenberg/{}{}'.format(gutenberg_id, file_type)
             cover_url = get_stored_file_url(self.file_bucket, cover_path)
+            cover_root = yaml_file.get('url').replace('ebooks', 'files')
+            cover_source_url = f"{cover_root}/{cover_data.get('image_path')}"
+            response = requests.head(cover_source_url, allow_redirects=True)
+            response.raise_for_status()
```

Comment:
We could skip this and add this step in the record file saver step.

Reply:
But i understand we may want to double check

Reply:
Currently, the file saver will fail if any of the parts fail to save, which I think makes sense, as the new ETL is designed to not save the records without the files. I first went about changing the file saver to not fail specifically for cover images, but opted not to add that logic in favor of keeping it in the specific ingest, mainly to avoid making a decision *now* that affects other future ingests without more knowledge.

However, happy to implement it there!